src/schemas/SettingsSchema.js

```diff
@@ -3,14 +3,16 @@ const graphql = require('graphql');
 module.exports = graphql.buildSchema(`
   type Query {
     sites(siteId: String): SiteCollection
-    twitterAccounts(siteId: String!): TwitterAccountCollection
+    twitterAccounts(siteId: String!, pageState: String, pageSize: Int): TwitterAccountConnection
```

Comment:
The reason why I didn't add paging to the `TwitterAccountCollection` type was because I read in the [docs](http://graphql.org/learn/pagination/): `That leads us to a problem; though; how do we get the cursor from the object? We wouldn't want cursor to live on the User type; it's a property of the connection, not of the object.`

I agree with you in that it would be best to not have a non-backwards compatible change, so I'll update the schema!